man/man8/zfs-program.8

```diff
@@ -364,6 +365,27 @@ Valid only for destroying snapshots.
 If set to true, and the snapshot has holds or clones, allows the snapshot to be
 marked for deferred deletion rather than failing.
 .Ed
+.It Em zfs.sync.inherit(dataset, property)
+Clears the specified property in the given dataset, causing it to be inherited
+from an ancestor, or restored to the default is no ancestor property is set.
+The
+.Ql zfs inherit -S
+property has not been implemented.
```

Comment:
`option` instead of `property` since this is a subcommand.

```diff
-.Ql zfs inherit -S
-property has not been implemented.
+.Nm zfs Cm inherit -S
+option has not been implemented.
```